openadapt/events.py

```diff
@@ -457,24 +498,25 @@ def get_merged_events(to_merge, state):
                         "children": children,
                     },
                 )
-                state["dt"] += (last_child.timestamp - first_child.timestamp)
+                state["dt"] += last_child.timestamp - first_child.timestamp
             merged_events.append(merged_event)
         return merged_events
 
     return merge_consecutive_action_events(
-        "keyboard", events, is_target_event, get_merged_events,
+        "keyboard",
+        events,
+        is_target_event,
+        get_merged_events,
     )
 
 
-def remove_redundant_mouse_move_events(events):
-    """Remove mouse move events that don't change the mouse position"""
-
+def remove_redundant_mouse_move_events(events: Any) -> list:
+    """Remove mouse move events that don't change the mouse position."""
 
-    def is_target_event(event, state):
+    def is_target_event(event: Any, state: Any) -> bool:
```

Comment:
`event: ActionEvent, state: dict[str, Any]`

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507